go/libkb/chain_link.go

```diff
@@ -1134,6 +1208,22 @@ func (c *ChainLink) GetSigchainV2TypeFromV2Shell() (SigchainV2Type, error) {
 	return c.unpacked.outerLinkV2.LinkType, nil
 }
 
+// Helper function for getting a ChainLink's type. If it is a v2 link (that may
```

Comment:
use Go-style commenting where you begin the comment with the name of the function.